programs/marinade-finance/src/instructions/delayed_unstake/claim.rs

```diff
@@ -111,8 +102,13 @@ impl<'info> Claim<'info> {
             return Err(MarinadeError::TicketNotReady.into());
         }
 
-        self.state.circulating_ticket_balance -= lamports;
-        self.state.circulating_ticket_count -= 1;
+        // If circulating_ticket_balance = sum(ticket.balance) is violated we can have a problem
```

Comment:
this just makes the subtraction harder to read. please don't add checkedsub, we have overflow-checks=true
This will never happen, and in any case "underflow error" is as good or better than "calculation failure"